lib/mini_sql/mysql/deserializer_cache.rb

```diff
@@ -39,18 +39,9 @@ def new_row_matrializer(result)
 
         Class.new do
           extend MiniSql::Decoratable
-          attr_accessor(*fields)
-
-          # AM serializer support
-          alias :read_attribute_for_serialization :send
+          include MiniSql::Result
```

Comment:
does this refactoring lead to any noticeable perf diff, due to the way methods are looked up?

Reply:
bench is below